dev/test/gapic_firestore_v1.ts

```diff
@@ -1259,7 +1259,6 @@ describe('v1.FirestoreClient', () => {
         new protos.google.firestore.v1.WriteRequest()
       );
       request.database = '';
-      const expectedHeaderRequestParams = 'database=';
```

Comment:
Please exclude the generated test files as the `node10` branch contains the more recent versions.

Reply:
done.